components/styles/index.js

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export const StyledMain = styled.main`
+  margin: 4rem auto;
+  padding: 0;
+
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  @media only screen and (max-width: 799px) {
+    width: 100vw;
+  }
+  @media only screen and (min-width: 800px) {
+    width: 640px;
+    left: 0;
+    right: 0;
+  }
+`;
+
+export const StyledLink = styled(Link)`
+  padding: 0.3rem 1rem;
+
+  display: flex;
```

Comment:
Are you sure you need the display: flex here? Ahh maybe if you have icons in it?

Reply:
Yes. that's the reason.  😄